M2/Macaulay2/d/common.d

```diff
@@ -45,10 +45,6 @@ export codePosition(c:Code):Position := ( -- TODO retire
     is f:Error                     do f.position
     );
 
-export setuppostfix(e:SymbolClosure,fn:unop):void := (
-     unopNameList = unopNameListCell(fn,e.symbol,unopNameList);
-     e.symbol.postfix = fn;
-     );
```

Comment:
I understand what this commit does, but was it necessary? I see the simplification, but also seems like not a bad idea to be able to differentiate postfix vs unary operators in the interpreter.